### ISSUE TITLE
Forge's new registry system is complaining about Cyclops Core and crashing the game.

### ISSUE THREAD

**Issue by Zarepheth** [CyclopsMC/CyclopsCore#53]
As the title says, the newer versions of Forge are breaking Cyclops Core.

Minecraft: 1.12
Forge: 14.21.02379
Cyclops Core: 0.10.6

```
[21:29:42] [main/FATAL] [FML]: The following problems were captured during this phase
[21:29:42] [main/ERROR] [FML]: Caught exception from cyclopscore (java.lang.NoClassDefFoundError: net/minecraftforge/event/AttachCapabilitiesEvent$Item)
```

```
Description: There was a severe problem during mod loading that has caused the game to fail

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Cyclops Core (cyclopscore)
Caused by: java.lang.NoClassDefFoundError: net/minecraftforge/event/AttachCapabilitiesEvent$Item
	at org.cyclops.cyclopscore.init.ModBase.constructCapabilityConstructorRegistry(ModBase.java:123)
	at org.cyclops.cyclopscore.init.ModBase.<init>(ModBase.java:87)
	at org.cyclops.cyclopscore.CyclopsCore.<init>(CyclopsCore.java:61)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at java.lang.Class.newInstance(Unknown Source)
	at net.minecraftforge.fml.common.ILanguageAdapter$JavaAdapter.getNewInstance(ILanguageAdapter.java:193)
	at net.minecraftforge.fml.common.FMLModContainer.constructMod(FMLModContainer.java:618)
	...
```
[Log File on paste.ee](https://paste.ee/p/rTBHJ)

**Comment by carstorm** [comment]
Can confirm this is also happening to me, believe its the same base issue:

Minecraft: 1.12
Forge: 14.21.02375
Cyclops Core: 0.10.6

pastebin: https://pastebin.com/TrbucE2x

**Comment by rubensworks** [comment]
Once a RB for Forge 1.12 has been released I will release new updates. The Forge API is a bit unstable atm.

**Comment by rubensworks** [comment]
Fixed in b6a7a9666b141537904ff8dc3325f31a7f9848f6, new 0.10.7 release is incoming.
